### DISCUSSION TITLE
twin.macro support?

### DISCUSSION THREAD

**Discussion by heygoran** [RyanClementsHax/tailwindcss-themer#27] [Q&A] [answered]
Hello,

Is it possible to use tailwindcss-themer with the twin.macro?

Regards,
Goran

**Comment by RyanClementsHax** [comment]
I'm not even sure what that library does to be honest. I currently have no plans to support it but I'm not opposed to contributions though :)

**Reply by FranciscoKloganB** [reply, marked as answer]
This is an old thread, but hopefully anyone in the future, might find this helpful...

`twin.macro` allows people to use `tailwindcss` classes in a `styled` component approach, using `styled-components`, `emotion` or even `stitches` (_warning: stitches is no longer maintained_).

Since `twin.macro` released their 3rd stable version (i.e., v3.0.0+), it technically supports all of native tailwind features (because the newest implementation no longer needs to manually reproduce every feature, instead if integrates directly with Tailwind APIs), consequently, it also supports all plugins that native tailwind supports. That means, this one in no different.

See here: https://github.com/ben-rogerson/twin.macro/releases/tag/3.0.0

**Reply by RyanClementsHax** [reply]
huzzah!
